core/templates/pages/contributor-dashboard-page/services/contribution-and-review.service.ts

```diff
@@ -170,6 +182,143 @@ export class ContributionAndReviewService {
     );
   }
 
+  async fetchTranslationSuggestionsAsync(
+      explorationId: string
+  ): Promise<FetchSuggestionsResponse> {
+    const explorationBackendResponse = await this.
+      readOnlyExplorationBackendApiService.fetchExplorationAsync(
+        explorationId, null);
+    return(
+      this.contributionAndReviewBackendApiService.
+        fetchSuggestionsAsync(
+          'REVIEWABLE_TRANSLATION_SUGGESTIONS',
+          null,
+          0,
+          AppConstants.SUGGESTIONS_SORT_KEY_DATE,
+          explorationId
+        ).then((fetchSuggestionsResponse) => {
+          const explorationBackendDict = this.explorationObjectFactory.
+          createBackendDictFromExplorationBackendResponse(
+            explorationBackendResponse);
+
+          const exploration: Exploration = this.explorationObjectFactory.
+            createFromBackendDict(explorationBackendDict);
+          const sortedTranslationCards = this.sortTranslationSuggestionsByState(
+            fetchSuggestionsResponse.suggestions,
+            exploration.getStates(),
+            explorationBackendDict.init_state_name
+          );
+          // eslint-disable-next-line
```

Comment:
What is this suppressing?

Reply:
We are suppressing lint/ts error caused due to use of any datatype here and elsewhere too.